packages/rgbpp/src/service.ts

```diff
@@ -0,0 +1 @@
+export * from '@rgbpp-sdk/btc';
```

Comment:
Same as above, this should be `@rgbpp-sdk/service`?

Reply:
Fixed